dbt-snowflake/src/dbt/adapters/snowflake/impl.py

```diff
@@ -432,15 +455,47 @@ def debug_query(self):
 
     @classmethod
     def _get_adapter_specific_run_info(cls, config: RelationConfig) -> Dict[str, Any]:
-        table_format: Optional[str] = None
-        if (
-            config
-            and hasattr(config, "_extra")
-            and (relation_format := config._extra.get("table_format"))
-        ):
-            table_format = relation_format
-
-        return {
-            "adapter_type": "snowflake",
-            "table_format": table_format,
+        # `config` is not a RelationConfig!
```

Comment:
what's this mean?

Reply:
We typed the input of this method as a `RelationConfig` in both the base and here. However, while troubleshooting this, I determined that the object that gets passed into this method is not actually a `RelationConfig` instance. I couldn't update the type annotation here without updating it in the base or `mypy` would raise an error. And I felt like that update is outside of the scope of this PR.